src/hooks/useActiveDescendantListFocus/index.js

```diff
@@ -123,6 +163,13 @@ function useActiveDescendantListFocus({
     }
   }, [visualFocusItemIndex]);
 
+  const onFocusCallback = useCallback(() => {
+    if (visualFocusItemIndex === -1 && defaultVisualFocusFirstItem) {
+      triggerByKeyboard.current = true;
```

Comment:
but what if focus is not by keyboard? it will draw you a focus when the focus trigger is a mouse... need to check it